numpy/linalg/linalg.py

```diff
@@ -135,7 +136,10 @@ def _linalgRealType(t):
 
 def _commonType(*arrays):
```

Comment:
I think this now crashes if the type of the array is `longdouble`

Reply:
@eric-wieser yes it will crash if the array type is longdouble. But the issue of longdouble is present regardless of the float16 issue. I was trying to resolve only the float16 issue. I can make the changes so as to support the longdouble datatype also, if needed. Should I work on the longdouble part?

Reply:
Nevermind, I hadn't realized it already threw an error for that case